fusor-ember-cli/app/controllers/assign-nodes.js

```diff
@@ -64,7 +64,7 @@ export default Ember.Controller.extend(DeploymentControllerMixin, {
     var params = this.get('model.plan.parameters');
     var self = this;
     this.get('model.plan.roles').forEach(function(role, index) {
-      count += self.getParamValue(role.get('countParameterName'), params);
+      count += parseInt(self.getParamValue(role.get('countParameterName'), params));
```

Comment:
it is generally highly recommended to pass in a radix (second argument) so that there is no confusion as to intended result - from Javascript: The Good Parts
```
If the first character of the string is 0, then the string is evaluated in base 8 instead of base 10. In base 8, 8 and 9 are not digits, so parseInt("08") and parseInt("09") produce 0 as their result. This error causes problems in programs that parse dates and times. Fortunately, parseInt can take a radix parameter, so that parseInt("08", 10) produces 8. I recommend that you always provide the radix parameter.
```